sys/console/full/syscfg.yml

```diff
@@ -43,15 +43,31 @@ syscfg.defs:
         value: 256
     CONSOLE_HISTORY_SIZE:
         description: >
-            Number of lines to be stored in console history.
-            Set to "0" to disable console history.
+            This is no longer used.
+            Equivalent settings:
+                CONSOLE_HISTORY: ram
+                CONSOLE_HISTORY_RAM_HISTORY_SIZE: <n>   <- previous value of CONSOLE_HISTORY_SIZE
         value: 0
-    CONSOLE_MAX_PROMPT_LEN:
+        deprecated: 1
```

Comment:
if this is not used (i.e. it does nothing as there is no new syscfg which "inherits" its value) then it should better be `defunct`. also I'd move it somewhere at the end of settings so it does not bother anyone

Reply:
moved to end and changed to defunct